client/v3/watch.go

```diff
@@ -1037,7 +1037,7 @@ func (pr *progressRequest) toPB() *pb.WatchRequest {
 
 func streamKeyFromCtx(ctx context.Context) string {
 	if md, ok := metadata.FromOutgoingContext(ctx); ok {
-		return fmt.Sprintf("%+v", md)
+		return fmt.Sprint(map[string][]string(md))
```

Comment:
let's try to keep the format consistent
```suggestion
		return fmt.Sprintf("%+v", map[string][]string(md))
```